### ISSUE TITLE
Focus indicator border fails 

### ISSUE THREAD

**Issue by ashleypeacock** [melissaegg/trauma-informed-design-resources#2]
### Issue
State: Partial fail. 
The blue, on the focus indicator, around the headings of the cards fails minimum colour contrast for [non-text components.](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html).
Given the border is slightly outside of the button, it's not a full fail under WCAG 2.1. 

<img width="341" alt="Screenshot 2024-07-24 at 01 42 43" src="https://github.com/user-attachments/assets/c18235d0-c5f3-4fc9-b289-549c41f6c703">

### Recommendations
* Thicken the border around the headings to a minimum 2 px to move from 'partial fix' to full fix.
* [Review WCAG 2.2 AAA Focus appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)

**Comment by melissaegg** [comment]
I *think* I fixed this. I added this code:  
*:focus {
            outline: 3px solid;
        }
